tests/test_other.py

```diff
@@ -4578,11 +4578,7 @@ def test(args=[]):
     test(['--bind'])
 
   def test_dashE_respect_dashO(self):
-    # issue #3365
```

Comment:
should the issue link be removed?

Reply:
Oh, heh, that's bad. I'll fix before I land.

Reply:
Although, it's not obvious what the issue has to do with this test.

Reply:
Looks like the #issue 3365 comments were left around in the landed form.

Reply:
Yes, I preserved the history, as those tests were added because of that issue.

Reply:
(That might have been a mistake at the time, I didn't investigate now...)